be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.pl.resx

```diff
@@ -144,6 +144,9 @@
   <data name="EmailIsConfirmedText" xml:space="preserve">
     <value>Konto Potwierdzone</value>
   </data>
+  <data name="EmailMarketingApprovalLabel" xml:space="preserve">
+    <value>Wyrażam zgodę na otrzymywanie informacji handlowych od ELTAP drogą elektroniczną, na wskazany przeze mnie adres e-mail</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP